packages/ceramic-cli/src/ceramic-logger-plugins.ts

```diff
@@ -14,10 +14,9 @@ import {
  */
 export class LogToFiles {
     /**
-     * Modifies `rootLogger` to append log messages to files named after components
-     * @notice If no component name is given 'default' will be included in the file name
+     * Modifies `rootLogger` to append log messages to files
      * @param rootLogger Root logger to use throughout the library
-     * @param loggerOptions Should include `component` name string and `logPath` string
+     * @param loggerOptions null
```

Comment:
should we add a description here?

Reply:
Yeah good idea. `loggerOptions` is not being used in this particular plugin so I'll add a note saying so there.